float64                     azimuth
string                      transcript